File: api/Proyecto_BK/Proyecto_BK.Common/Models/FacturaViewModel.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Proyecto_BK.Common.Models
{
    public class FacturaViewModel
    {
        public DateTime? Fact_Fecha { get; set; }
        public decimal? Fact_Total { get; set; }
        public string Clie_Identidad { get; set; }
        public string Clie_Nombre { get; set; }
        public int? Fact_Id { get; set; }
        public int? FaDe_Cantidad { get; set; }
        public int? FaDe_Subtotal { get; set; }
        public string FaDe_Ident { get; set; }
        public int? FaDe_ProdId { get; set; }
        public string Producto { get; set; }
        public string Precio { get; set; }
        public string Cantidad { get; set; }
        public string Categoria { get; set; } 
        public string Total { get; set; }
        [NotMapped]
        public string Alim_Descripcion { get; set; }
        [NotMapped]
        public string Alim_Precio { get; set; }
        [NotMapped]
        public string Alim_Imagen { get; set; }

        [NotMapped]
        public string Bebi_Descripcion { get; set; }
        [NotMapped]
        public string Bebi_Precio { get; set; }
        [NotMapped]
        public string Bebi_Imagen { get; set; }

        [NotMapped]
        public string Comp_Descripcion { get; set; }
        [NotMapped]
        public string Comp_Precio { get; set; }
        [NotMapped]
        public string Comp_Imagen { get; set; }

        [NotMapped]
        public string Post_Descripcion { get; set; }
        [NotMapped]
        public string Post_Precio { get; set; }
        [NotMapped]
        public string Post_Imagen { get; set; }
        [NotMapped]
        public string Sucu_Descripcion { get; set; }
        [NotMapped]
        public string Empleado { get; set; }


        [NotMapped]
        public string CantidadVentas { get; set; }

    }
}

File: app/plantilla/src/app/demo/models/FacturaViewModel.ts
import { DecimalPipe } from "@angular/common";

export class Factura {
     Sucu_Id?:number;
     Sucu_Descripcion?:string; 
     Empl_Id ?:number;
     Empl_Nombre ?:string;
     Fact_Fecha ?:Date;
     Fact_Total ?:DecimalPipe;
     Clie_Id?:number;
     Clie_Nombre?:String;
     Clie_Identidad?:string;
}

export class FacturaDetalle {
     clie_Nombre?:string;
     FaDe_Ident?:number;
     FaDe_Cantidad?:number;
     FaDe_Subtotal?:number;
     FaDe_ProdId?:number;
     fact_Id?:number;
     producto?:string;
     precio?:string;
     cantidad?:string; 
     categoria?:string;
     total?:string;
 }
 
 export class FacturaDetalleEnviar {
    Fact_Id?:number;
    FaDe_Ident?:number;
    FaDe_Cantidad?:number;
    FaDe_Subtotal?:number;
    FaDe_ProdId?:number;
    Prod_Producto?:string;
    Fact_Fecha ?:Date;
    Fact_Total ?:DecimalPipe;
    Clie_Id?:number;
    Clie_Nombre?:String;
    Clie_Identidad?:string;
 }
 

 export class FacturaEnviar {
     Fact_Id:number;
     FaDe_Ident?:number;
     Sucu_Id?:number;
     Empl_Id ?:number;
     Empl_Nombre ?:string;
     Fact_Fecha ?:Date;
     Fact_Total ?:DecimalPipe;
     Clie_Nombre?:String;
     Clie_Identidad?:string;
     FaDe_ProdId?:number;
     Prod_Nombre?:String;
     FaDe_Cantidad?:number; 
 }
 
export class Complemento{
Comp_Id?:number;
Comp_Descripcion?:string;
}

export class FacturaEliminar{
    FaDe_Ident?:string;
    Prod_Nombre?:String;
    Fact_Id:number;
    }

export class Postre{
Post_Id?:number;
Post_Descripcion?:string;
}


export class Bebida{
    Bebi_Id?:number;
    Bebi_Descripcion?:string;     
}


export class Combo{
    Comb_Id?:number;
    Comb_Descripcion?:string;
}


export class Paquete{
   Paqe_Id?:number;
   Paqe_Descripcion?:string;
}
